Ports/p7zip/patches/fixes.patch

```diff
@@ -19,7 +19,7 @@ index 469e325..32388ac 100644
  )
  
 +IF(SERENITYOS)
-+  TARGET_LINK_LIBRARIES(7za ${CMAKE_THREAD_LIBS_INIT} dl iconv)
++  TARGET_LINK_LIBRARIES(7za ${CMAKE_THREAD_LIBS_INIT} dl iconv pthread)
```

Comment:
```suggestion
+  TARGET_LINK_LIBRARIES(7za Threads::Threads dl iconv)
```

The port's `configopts` should also be edited so `THREADS_PREFER_PTHREAD_FLAG` is set to true to use the `-pthread` flag. That is, unless using that instead of `-lpthread` is somehow an issue.

`Threads::Threads` was [added in CMake 3.1](https://cmake.org/cmake/help/latest/module/FindThreads.html#imported-targets), but that's really only a problem if you want to build this with 2.8.12 or 3.0 :^)

Reply:
> THREADS_PREFER_PTHREAD_FLAG
> [...]
> This variable has no effect if the system libraries provide the thread functions, [...]

Reply:
Oh, serenity does that? Then linking to `CMAKE_THREAD_LIBS_INIT` is pointless anyway, considering the rest of the paragraph.

Reply:
It doesn't, but CMake believes that it does (due to the `CMAKE_TRY_COMPILE_TARGET_TYPE` breakage, which is half of the reason why this PR exists), so supplying that flag and linking against `Threads::Threads` does not make a difference, since it ends up empty anyways.